exports.js

```diff
@@ -7,7 +7,7 @@ export default {
 
   // SendbirdProvider
   SendbirdProvider: 'src/lib/Sendbird.jsx',
-  sendBirdSelectors: 'src/lib/selectors.js',
+  sendBirdSelectors: 'src/lib/selectors.js',// We must to change this to sendbirdSelectors
```

Comment:
rename  it and add to change-log and migration guide

Reply:
maybe seperate PR is better

Reply:
yeah